splunk/src/main/java/com/splunk/Xml.java

```diff
@@ -52,6 +52,10 @@ public static Document parse(InputStream input, boolean silent) {
         try {
             DocumentBuilderFactory factory =
                 DocumentBuilderFactory.newInstance();
+            factory.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
+            factory.setExpandEntityReferences(false);
+            factory.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
+            factory.setFeature("http://xml.org/sax/features/external-general-entities", false);
             factory.setNamespaceAware(false);
```

Comment:
Also let's use https if possible here once I understand the intent of the change

Reply:
Using https in feature URLs are not available when tests are executed. So all http URLs are correct.